rm-common-debugging.md

```diff
@@ -0,0 +1,29 @@
+# List of Common Debugging
+
+ * Cannot use VM on windows 10 and Docker at the same time
+ * Cannot VPN into network and use Docker (nginex crashes) when working remotely
+ * Cannot execute 'docker exec' command in git bash
+
+ ## Debugging '...\r'
```

Comment:
## Debugging '...\r'
You might have an issue with windows vs linux line endings if you see errors with '..\r'. Here's how we fixed this in the past:

^I think this needs to be a little more explicit.